templates/garbage-collector.yml

```
@@ -39,6 +39,9 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ .Values | toYaml | sha256sum }}
+        {{- with .Values.deployment.garbage-collector.template_annotations }}
```

```suggestion
        {{- with index .Values "deployment.garbage-collector.template_annotations" }}
```
the '-' in garbage-collector breaks the template otherwise